client/.eslintrc.json

```diff
@@ -0,0 +1,110 @@
+{
+    "extends": ["eslint:recommended", "plugin:react/recommended", "prettier"],
+    "parserOptions": {
+        "ecmaFeatures": {
+            "jsx": true
+        },
+        "ecmaVersion": 2020,
+        "sourceType": "module"
+    },
+    "settings": {
+        "react": {
+            "version": "detect"
+        }
+    },
+    "env": {
+        "browser": true,
+        "jest": true,
+        "node": true
+    },
+    "rules": {
+        "array-bracket-newline": "error",
+        "camelcase": "error",
```

Comment:
why no camelCase?

Reply:
It actually means only camelCasing is accepted...

Reply:
alright got it

__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Tags:** Efficient